src/Domain/CoordinateMaps/Wedge3D.hpp

```diff
@@ -25,16 +25,14 @@ namespace CoordinateMaps {
  *
  * \details The mapping that goes from a reference cube to a three-dimensional
  *  wedge centered on a coordinate axis covering a volume between an inner
- *  surface and outer surface. The inner surface can be given a curvature
+ *  surface and outer surface. Each surface can be given a curvature
  *  between flat (a sphericity of 0) or spherical (a sphericity of 1).
- *  The sphericity of the outer surface is currently constrained to have a
- *  value of 1.
  *
  *  The first two logical coordinates correspond to the two angular coordinates,
  *  and the third to the radial coordinate.
  *
  *  The Wedge3D map is constructed by linearly interpolating between a bulged
- *  face of radius `radius_of_inner_surface` to a spherical face of
+ *  face of radius `radius_of_inner_surface` to a bulged face of
  *  radius `radius_of_outer_surface`, where the radius of the bulged face
```

Comment:
I think it is clearer to say "the radius of each bulged face", as this is more consistent with having more than one bulged face.